Cargo.toml

```diff
@@ -36,6 +36,8 @@ feat_os_unix_fuchsia = [
   "chgrp",
   "chmod",
   "chown",
+  "chroot",
+  "df",
```

Comment:
This only works for unix fuschia?

Reply:
This should likely be relocated to the Tier-1 feature set, "feat_common", or if it should build/run everywhere, into the "feat_common_core" set.